script-library/common-debian.sh

```diff
@@ -107,7 +109,10 @@ if [ "${PACKAGES_ALREADY_INSTALLED}" != "true" ]; then
         sudo \
         ncdu \
         man-db \
-        strace"
+        strace
```

Comment:
I think you need a `\` here so that it's all the same command

```suggestion
        strace \
```

Reply:
Done.